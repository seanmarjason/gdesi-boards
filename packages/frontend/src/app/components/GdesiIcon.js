import * as React from 'react';
import SvgIcon from '@mui/material/SvgIcon';
import Box from '@mui/material/Box';

export default function GdesiIcon() {
  return (
    <Box
      component="img"
      sx={{
        height: 25,
        width: 100,
        maxHeight: { xs: 233, md: 167 },
        maxWidth: { xs: 350, md: 250 },
      }}
      alt="The house from the offer."
      src="desi-sm-to2.png"
    />
  );
}
