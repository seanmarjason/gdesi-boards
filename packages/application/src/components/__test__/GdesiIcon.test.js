import { render } from '@testing-library/react'
import GdesiIcon from '../GdesiIcon';

it('GdesiIcon renders correctly', () => {
  const { container } = render(<GdesiIcon />)
  expect(container).toMatchSnapshot()
})
