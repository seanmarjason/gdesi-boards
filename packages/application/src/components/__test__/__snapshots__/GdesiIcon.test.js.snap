// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GdesiIcon renders correctly 1`] = `
<div>
  <img
    alt="The house from the offer."
    class="MuiBox-root css-ndi7rl"
    src="desi-sm-to2.png"
  />
</div>
`;
